src/liftwing_api/models/revertrisk.py

```diff
@@ -0,0 +1,44 @@
+import requests
+import json
+from liftwing_model import LiftwingModel
+
+class RevertRiskAPIModel(LiftwingModel):
+    def __init__(self, base_url="https://api.wikimedia.org/service/lw/inference/v1/models/{language}-reverted:predict"):
```

Comment:
This is not the url for the revertrisk model 
```
"https://api.wikimedia.org/service/lw/inference/v1/models/{language}-reverted:predict"
```
This is the link to the relevant documentation for the revertrisk language agnostic model 
[https://api.wikimedia.org/wiki/Lift_Wing_API/Reference/Get_reverted_risk_language_agnostic_prediction](https://api.wikimedia.org/wiki/Lift_Wing_API/Reference/Get_reverted_risk_language_agnostic_prediction)